lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactoryTest.java

```diff
@@ -139,14 +147,75 @@ void shouldFailWhenKbvItemStatusIsAnyOtherValue() throws JsonProcessingException
 
     @Nested
     class EvidenceCheckDetails {
+
+        @Test
+        @DisplayName(
+                "should use the default verification if verification score from session item is 0")
+        void shouldHaveAVerificationScoreOfZero() throws JsonProcessingException {
+            KBVItem kbvItem = getKbvItem();
+            UUID sessionId = UUID.randomUUID();
+            kbvItem.setSessionId(sessionId);
+            kbvItem.setStatus("authenticated");
+            kbvItem.setQuestionAnswerResultSummary(getKbvQuestionAnswerSummary(3, 3, 0));
+            setKbvItemQuestionState(kbvItem, "First", "Second", "Third");
+
+            SessionItem sessionItem = new SessionItem();
+            EvidenceRequest evidenceRequest = new EvidenceRequest();
+            evidenceRequest.setVerificationScore(0);
+            sessionItem.setEvidenceRequest(evidenceRequest);
```

Comment:
This is sort of fine, but we should also test when there is no evidenceRequest i.e. evidenceRequest is null